frontend/packages/ux-editor/src/utils/objectUtils.ts

```diff
@@ -0,0 +1,12 @@
+import { KeyValuePairs } from 'app-shared/types/KeyValuePairs';
+
+/**
+ * Maps an array of objects to a key-value pair object, where the key is the value of the property.
+ * Requires that the property value is unique for each object.
```

Comment:
Should there be a test checking that the function does not crash when the values are not unique?

And I also struggle with the "ordlyd" here; first I read 'property value' as the value of the property; meaning 'id' in the tests, but I understand that it is the corresponding value of the property it refers to. Maybe difficult not to be ambiguous tho...

Reply:
I think it's better to throw a specific error message, since this is not supposed to happen. I'll fix that. I'll have a look at the comment as well 🙂